### PR TITLE
registry: Show text and blank slate when various views are empty

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

File: pkg/kubernetes/views/registry-dashboard-page.html [hunk 1]
```diff
@@ -1,11 +1,24 @@
-<div class="container-cards-pf dashboard-cards">
+<div class="blank-slate-pf" ng-show="!filter.namespaces().length">
+    <div class="blank-slate-pf-icon">
+        <i class="pficon pficon-add-circle-o"></i>
+    </div>
+    <h1>Welcome to the Image Registry</h1>
```
Comment (petervo) [inline pkg/kubernetes/views/registry-dashboard-page.html]: translatable

File: pkg/kubernetes/views/registry-dashboard-page.html [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+    <p>In order to begin pushing images to the registry, you need to create a project.</p>
```
Comment (petervo) [inline pkg/kubernetes/views/registry-dashboard-page.html]: translatable

**Comment by petervo** [comment]
A few test failures that may need to be addressed.

**Comment by dperpeet** [comment]
needs rebase

**Pull request closed without merging**